src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/CaseDataController.java

```diff
@@ -118,6 +118,7 @@ public ResponseEntity<AboutToStartOrSubmitCallbackResponse> setContestedPaperCas
         log.info("Setting default values for contested paper case journey.");
         validateCaseData(callbackRequest);
         setOrganisationPolicyForNewPaperCase(callbackRequest.getCaseDetails());
+        callbackRequest.getCaseDetails().getData().put(CIVIL_PARTNERSHIP, NO_VALUE);
```

Comment:
better to use     caseData.putIfAbsent(CIVIL_PARTNERSHIP, NO_VALUE);

Reply:
I've applied this code change Rishi, thanks